content/en/docs/concepts/architecture/garbage-collection.md

```diff
@@ -37,6 +37,24 @@ to the labels, each `EndpointSlice` that is managed on behalf of a Service has
 an owner reference. Owner references help different parts of Kubernetes avoid
 interfering with objects they don’t control.
 
+{{< note >}}
```

Comment:
I don't have a good sense for whether this makes more sense on this page or on /docs/concepts/overview/working-with-objects/owners-dependents/

the bit about cross-namespace and cluster→namespaced references not being allowed is really important … it might bear repeating on both pages or mentioning/linking

Reply:
Since the note is highly important, I've added the note to /docs/concepts/overview/working-with-objects/owners-dependents.md also